### PR TITLE
PMM-6820 add --wait flag to pmm-admin status command

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: agentlocal/agentlocal.go [hunk 1]
```diff
@@ -55,6 +55,7 @@ const (
 
 // ErrNotSetUp is returned by GetStatus when pmm-agent is running, but not set up.
 var ErrNotSetUp = fmt.Errorf("pmm-agent is running, but not set up")
+var ErrNotConnected = fmt.Errorf("pmm-agent is not connected to PMM server")
```
Comment (askomorokhov) [inline agentlocal/agentlocal.go]: Looks like it is used only once.

File: commands/status.go [hunk 2]
```diff
@@ -102,3 +118,7 @@ var (
 	Status  = new(statusCommand)
 	StatusC = kingpin.Command("status", "Show information about local pmm-agent")
 )
+
+func init() {
+	StatusC.Flag("wait", "Timeout to get success response from pmm-agent").DurationVar(&Status.timeout)
```
Comment (askomorokhov) [inline commands/status.go]: Timeout looks like a not proper word here. `s/timeout/deadline/g`?
Maybe better deadline? `Time deadline to get a success response.`
@PaulJacobs-percona , could you please suggest a better word for: a maximum time to wait on the response?

**Review by askomorokhov — APPROVED** [review]
Looks good to me with comments.

File: agentlocal/agentlocal.go (hunk 1 above)
Comment (BupycHuk) [reply agentlocal/agentlocal.go]: Yes, it is. But it's extracted to make it visible.

File: commands/status.go (hunk 2 above)
Comment (ghost) [reply commands/status.go]: True that 'timeout' isn't a dictionary word (well, not UK), but it's familiar with command-line junkies and has a generally-accepted meaning. But my Mac dictionary (american english) says "2 (timeout) Computing a cancellation or cessation that automatically occurs when a predefined interval of time has passed without a certain event occurring."

**Review by percona-csalguero — APPROVED** [review]

**Pull request merged**